x/posts/internal/types/models/post_response_test.go

```diff
@@ -64,9 +64,9 @@ func TestPostQueryResponse_MarshalJSON(t *testing.T) {
 		"dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1",
 	}
 
-	reactionsResponses := []models.ReactionQueryResponse{
-		models.NewReactionQueryResponse("https://example.com/like", ":like:", liker),
-		models.NewReactionQueryResponse("👍", ":+1:", otherLiker),
+	reactionsResponses := []models.PostReaction{
+		models.NewPostReaction("https://example.com/like", ":like:", liker),
+		models.NewPostReaction("👍", ":+1:", otherLiker),
```

Comment:
These lines are wrong, the `NewPostReaction` first parameter is the shortcode, and then the value. Not the opposite: 

```suggestion
		models.NewPostReaction(":like:", "https://example.com/like", liker),
		models.NewPostReaction(":+1:", "👍", otherLiker),
```